WordPress/Sniffs/WP/EnqueuedResourcesSniff.php

```diff
@@ -45,7 +47,7 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
 		$tokens = $phpcsFile->getTokens();
 		$token  = $tokens[ $stackPtr ];
 
-		if ( preg_match( '#rel=[\'"]?stylesheet[\'"]?#', $token['content'] ) > 0 ) {
+		if ( preg_match( '#rel=\\\\?[\'"]?stylesheet\\\\?[\'"]?#', $token['content'] ) > 0 ) {
```

Comment:
Yup, but it works.... :heart_eyes: 

![escaping](https://cloud.githubusercontent.com/assets/663378/24132941/79ad9030-0df9-11e7-8cea-e4530352d220.png)